gcb/release.yaml

```diff
@@ -7,12 +7,17 @@ secrets:
 
 steps:
 - name: gcr.io/cloud-builders/git
+  dir: "go/src/k8s.io"
   args:
   - "clone"
-  - "https://github.com/kubernetes/release"
+  - "--branch=${_RELEASE_TOOL_BRANCH}"
+  - "${_RELEASE_TOOL_REPO}"
 
 - name: gcr.io/$PROJECT_ID/k8s-cloud-builder
-  dir: release
+  dir: "go/src/k8s.io/release"
+  env:
+  - "RELEASE_TOOL_REPO=${_RELEASE_TOOL_REPO}"
+  - "RELEASE_TOOL_BRANCH=${_RELEASE_TOOL_BRANCH}"
```

Comment:
are these still necessary in the future?

Reply:
TL;DR: We need this (at least for now) for every `anago` call

We need this every time we call `anago` and want the to support alternative repos/branches/refs ... And I don't see why we wouldn't want this to be configurable, esp. now that we need to touch the release tooling and some major refactoring is about to happen soon.

They are necessary right now, as `anago` pretty much on every call does check if the repo is in the state it is supposed to be. Besides the fact that this check does not make too much sense on GCB (the first step on GCB is to checkout that repo from that branch and to that ref anyway), I didn't want to touch that right now and keep the current behaviour. On the long run we will probably remove that check anyway.

Reply:
Thanks for the answer! I agree that we need this option for refactoring, let's keep it.

I understand the git ref check is for the part run locally, so not sure about if we want to remove it in the future.